kuma/spam/akismet.py

```diff
@@ -0,0 +1,174 @@
+import urlparse
+
+from constance import config
+from django.conf import settings
+from requests.packages.urllib3.util import Retry
+from requests.adapters import HTTPAdapter
+from requests import Session, exceptions
+
+
+class AkismetError(Exception):
+    """
+    A custom exception that takes the requests' response, its status code
+    and an optional debug help directly from Akismet.
+    """
+    def __init__(self, response, status_code, debug_help):
+        self.response = response
+        self.status_code = status_code
+        self.debug_help = debug_help
+
+    def __str__(self):
+        return ('%s response, debug help: %s, full response: %s' %
+                (self.status_code, self.debug_help, self.response.text))
+
+
+class Akismet(object):
+    """
+    The Akismet client implementation wrapping a request session and
+    having API specific methods for easy spam and ham handling.
+    """
+    submission_success = 'Thanks for making the web a better place.'
```

Comment:
Could you add a comment that this is the [documented response](http://akismet.com/development/api/#submit-spam), so that an enthusiastic future dev doesn't localize it?